pages/blog/[slug].tsx

```diff
@@ -116,11 +119,19 @@ export const getStaticProps: GetStaticProps = async function({
     return { props: {} } // will render the 404 error
   }
 
-  return {
-    props: {
-      ...previewProps.props,
-      siteConfig: { title: siteConfig.title },
-    },
+  try {
+    const stats = fs.statSync(path.resolve(`./content/blog/${slug}.md`))
+    const lastModified = formatDate(stats.mtime)
+
+    return {
+      props: {
+        lastModified,
+        ...previewProps.props,
+        siteConfig: { title: siteConfig.title },
+      },
+    }
+  } catch (error) {
+    throw error
```

Comment:
Let's remove the `try {} catch () {}`

If the `catch` just throws the error then it's not really doing anything and it can just be thrown